translations/en.json

```
@@ -2311,19 +2311,19 @@
     "oct": "Oct",
     "nov": "Nov",
     "dec": "Dec",
-    "channel-muted-for-15-minutes": "Channel muted for 15 minutes\n (until {{duration}}) ",
-    "channel-muted-for-1-hour": "Channel muted for 1 hour\n (until {{duration}}) ",
-    "channel-muted-for-8-hours": "Channel muted for 8 hours\n (until {{duration}}) ",
-    "channel-muted-for-1-week": "Channel muted for 1 week \n (until {{duration}}) ",
-    "channel-muted-till-unmuted": "Channel muted till unmuted\n (until {{duration}}) ",
-    "chat-muted-for-15-minutes": "Chat muted for 15 minutes\n (until {{duration}}) ",
-    "chat-muted-for-1-hour": "Chat muted for 1 hour\n (until {{duration}}) ",
-    "chat-muted-for-8-hours": "Chat muted for 8 hours\n (until {{duration}}) ",
-    "chat-muted-for-1-week": "Chat muted for 1 week\n (until {{duration}}) ",
-    "chat-muted-till-unmuted": "Chat muted till unmuted\n (until {{duration}}) ",
+    "channel-muted-for-15-minutes": "Channel muted for 15 minutes\n(until {{duration}})",
+    "channel-muted-for-1-hour": "Channel muted for 1 hour\n(until {{duration}})",
+    "channel-muted-for-8-hours": "Channel muted for 8 hours\n(until {{duration}})",
+    "channel-muted-for-1-week": "Channel muted for 1 week \n(until {{duration}})",
+    "channel-muted-till-unmuted": "Channel muted till unmuted\n(until {{duration}})",
+    "chat-muted-for-15-minutes": "Chat muted for 15 minutes\n(until {{duration}})",
+    "chat-muted-for-1-hour": "Chat muted for 1 hour\n(until {{duration}})",
+    "chat-muted-for-8-hours": "Chat muted for 8 hours\n(until {{duration}})",
+    "chat-muted-for-1-week": "Chat muted for 1 week\n(until {{duration}})",
+    "chat-muted-till-unmuted": "Chat muted till unmuted\n(until {{duration}})",
```

```suggestion
    "channel-muted-for-15-minutes": "Channel muted for 15 minutes \n(until {{duration}})",
    "channel-muted-for-1-hour": "Channel muted for 1 hour \n(until {{duration}})",
    "channel-muted-for-8-hours": "Channel muted for 8 hours \n(until {{duration}})",
    "channel-muted-for-1-week": "Channel muted for 1 week \n(until {{duration}})",
    "channel-muted-till-unmuted": "Channel muted till unmuted \n(until {{duration}})",
    "chat-muted-for-15-minutes": "Chat muted for 15 minutes \n(until {{duration}})",
    "chat-muted-for-1-hour": "Chat muted for 1 hour \n(until {{duration}})",
    "chat-muted-for-8-hours": "Chat muted for 8 hours \n(until {{duration}})",
    "chat-muted-for-1-week": "Chat muted for 1 week \n(until {{duration}})",
    "chat-muted-till-unmuted": "Chat muted till unmuted \n(until {{duration}})",
```

Having a space before the new line makes it more legible